kitsune/search/tasks.py

```diff
@@ -121,7 +121,7 @@ def index_chunk_task(write_index, batch_id, chunk):
 
     except Exception:
         rec.text = u'{0}: Errored out {1} {2}'.format(
-            rec.text, sys.exc_type, sys.exc_value)
+            rec.text, sys.exc_type, sys.exc_value)[:255]  # Truncate at 255 chars.
```

Comment:
If you truncate this this way, then you're going to be losing important parts of the error. Any idea how much over 255 the text is? Should we send the error in an email instead?